assemble/bin/accumulo-cluster

```diff
@@ -340,10 +340,16 @@ function main() {
   accumulo_cmd="${bin}/accumulo"
   SSH='ssh -qnf -o ConnectTimeout=2'
 
+  manager_file="managers"
+  if [ ! -f "$conf/$manager_file" -a -f "$conf/masters" ]; then
```

Comment:
Prefer bash double square brackets for consistency and to avoid use (or apparent use) of external process `/usr/bin/\[`.

```suggestion
  if [[ ! -f "$conf/$manager_file" -a -f "$conf/masters" ]]; then
```